File: 프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.js
function solution(babbling) {
    const arr = ['aya', 'ye', 'woo', 'ma'];
    let answer = babbling.reduce((pre, cur) => {
        arr.forEach((value) => {
            cur = cur.replace(value, ' ');
        })
        if(cur.trim() === ''){
            return pre + 1;
        }
        return pre;
    }, 0);
    return answer;
}

File: 프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.py
def solution(babbling):
    words = ["aya", "ye", "woo", "ma"]
    answer = 0
    
    for word in babbling:
        for w in words:
            word = word.replace(w, ' ')
        if word.strip() == '':
            answer += 1
    return answer